game/src/GameMenu.tscn

```diff
@@ -38,5 +38,4 @@ alignment = 2
 [node name="LocaleButton" parent="HBox" instance=ExtResource("4_jno35")]
 layout_mode = 2
 
-[connection signal="options_button_pressed" from="MainMenu" to="." method="_on_main_menu_options_button_pressed"]
```

Comment:
This would prevent the OptionsMenu from ever showing if you clicked the options button. Pretty sure this should be reverted.

Reply:
should be fixed in dba52cd, I wonder why it did that